modules/core/test/v2/unit/coins/btc.ts

```diff
@@ -413,36 +413,48 @@ describe('BTC:', function() {
       });
 
       describe('failure', () => {
-        it('should fail for invalid transaction hexes', () => {
-          (() => coin.explainTransaction())
-          .should.throw('invalid transaction hex, must be a valid hex string');
-
-          (() => coin.explainTransaction({
-            txHex: ''
-          })).should.throw('invalid transaction hex, must be a valid hex string');
+        it('should fail for invalid transaction hexes', co(function *() {
+          try {
+            yield coin.explainTransaction();
```

Comment:
`yield coin.explainTransaction().should.be.rejectedWith('invalid ...');` is a little cleaner

Reply:
good one, changed.